src/components/select/select.scss

```diff
@@ -228,3 +228,41 @@ md-optgroup {
     border: 1px solid #fff;
   }
 }
+
+md-select-menu[multiple] {
+    md-option {
+        height: $select-option-height;
+        padding: 0 $select-option-padding 0 ($select-option-padding / 2);
+        padding-left: ($select-option-padding * 2) + ($select-option-padding / 2);
+
+        &:before {
```

Comment:
Agreed- copying is a pain for maintenance. Have you explored refactoring the checkbox css such that you can use the same classes directly?